packages/kbn-search-connectors/types/native_connectors.ts

```diff
@@ -776,13 +776,111 @@ export const NATIVE_CONNECTOR_DEFINITIONS: Record<string, NativeConnector | unde
         validations: [],
         value: '',
       },
+      use_domain_wide_delegation_for_sync: {
+        default_value: null,
+        depends_on: [],
+        display: DisplayType.TOGGLE,
+        label: i18n.translate(
+          'searchConnectors.nativeConnectors.gdrive.useDomainWideDelegation.label',
+          {
+            defaultMessage: 'Use domain-wide delegation for data sync',
+          }
+        ),
+        options: [],
+        order: 2,
+        required: true,
```

Comment:
I'm not sure these fields should be 'required' (doesn't really make sense for a boolean anyway).

Reply:
Hmm I followed a pattern from `use_document_level_security` and `use_text_extraction_service` which both are toggles and are required

Reply:
Probably doesn't affect things too much anyway

Reply:
TY for ship-it! By default all the fields in connector configuration are `required` unless specified otherwise ([source](https://github.com/elastic/connectors/blob/main/connectors/source.py#L51)) - therefore, I prefer to keep the original config as it was already manually tested